cpp/ql/test/query-tests/Critical/NewFree/NewFreeMismatch.expected

```diff
@@ -1,7 +1,5 @@
-| test2.cpp:19:3:19:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:18:10:18:14 | new | new |
-| test2.cpp:26:3:26:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:25:7:25:11 | new | new |
-| test2.cpp:26:3:26:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:25:7:25:11 | new | new |
-| test2.cpp:30:3:30:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:29:7:29:18 | new | new |
+| test2.cpp:19:3:19:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:18:12:18:18 | new | new |
+| test2.cpp:26:3:26:6 | call to free | There is a new/free mismatch between this free and the corresponding $@. | test2.cpp:25:7:25:13 | new | new |
```

Comment:
I'm not sure why these locations moved slightly -  did you rebuild the extractor between test runs?

Reply:
I can't explain that. I'll re-trigger the CI and merge if it's successful.

Reply:
I'm unlikely to have rebuilt the extractor, but it's likely I generated the first set of results, then tweaked the test code slightly and forgot to re-run the tests until the next commit.  (and if the tests passed on here I'm not worried)